examples/scripts/README.rst

```diff
@@ -41,6 +41,8 @@ Computed Tomography
       CT Training and Reconstructions with UNet
    `ct_projector_comparison.py <ct_projector_comparison.py>`_
       X-ray Transform Comparison
```

Comment:
Perhaps "X-ray Transform Comparison in 2D"? Also, it may be sligtly more accurate to phrased as "2D X-ray Transform Comparison" etc.

Reply:
agreed

Reply:
Done.